tests/installation/src/__tests__/extended/security-systems/convenience/acf2.ts

```diff
@@ -13,7 +13,7 @@ import {
   installAndVerifyConvenienceBuild,
   showZoweRuntimeLogs,
 } from '../../../../utils';
-import { TEST_TIMEOUT_CONVENIENCE_BUILD } from '../../../../constants';
+import {SECURITY_SYSTEM_ACF2, TEST_TIMEOUT_CONVENIENCE_BUILD} from '../../../../constants';
```

Comment:
You seem to be switching between `const security_system = identifySecuritySystem();` and hardcoding. I don't mind which you use, but it would be good to have consistency. If you are adding multiple imports then the linting style followed is to separate each import on a new line:
```
import { 
  TEST_TIMEOUT_CONVENIENCE_BUILD, 
  SECURITY_SYSTEM_ACF2,
} from '../../../../constants';
```
These issues should be resolved in all the test files